### PR TITLE
bpo-47081: Replace "qualifiers" with "quantifiers" in the re module documentation

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
https://bugs.python.org/issue47081

### CODE REVIEW

File: Doc/howto/regex.rst [hunk 1]
```diff
@@ -230,13 +230,13 @@ while ``+`` requires at least *one* occurrence.  To use a similar example,
 ``ca+t`` will match ``'cat'`` (1 ``'a'``), ``'caaat'`` (3 ``'a'``\ s), but won't
 match ``'ct'``.
 
-There are two more repeating qualifiers.  The question mark character, ``?``,
+There are two more repeating quantifiers.  The question mark character, ``?``,
```
Comment (serhiy-storchaka) [inline Doc/howto/regex.rst]:
```suggestion
There are two more repeating operators or quantifiers.  The question mark character, ``?``,
```

File: Doc/howto/regex.rst [hunk 2]
```diff
@@ -230,13 +230,13 @@ while ``+`` requires at least *one* occurrence.  To use a similar example,
...
 matches either once or zero times; you can think of it as marking something as
 being optional.  For example, ``home-?brew`` matches either ``'homebrew'`` or
 ``'home-brew'``.
 
-The most complicated repeated qualifier is ``{m,n}``, where *m* and *n* are
-decimal integers.  This qualifier means there must be at least *m* repetitions,
+The most complicated repeated quantifier is ``{m,n}``, where *m* and *n* are
```
Comment (serhiy-storchaka) [inline Doc/howto/regex.rst]:
```suggestion
The most complicated quantifier is ``{m,n}``, where *m* and *n* are
```

File: Doc/howto/regex.rst [hunk 3]
```diff
@@ -803,7 +803,7 @@ which matches the header's value.
 Groups are marked by the ``'('``, ``')'`` metacharacters. ``'('`` and ``')'``
 have much the same meaning as they do in mathematical expressions; they group
 together the expressions contained inside them, and you can repeat the contents
-of a group with a repeating qualifier, such as ``*``, ``+``, ``?``, or
+of a group with a repeating quantifier, such as ``*``, ``+``, ``?``, or
```
Comment (serhiy-storchaka) [inline Doc/howto/regex.rst]:
```suggestion
of a group with a quantifier, such as ``*``, ``+``, ``?``, or
```

File: Doc/library/re.rst [hunk 4]
```diff
@@ -87,7 +87,7 @@ Some characters, like ``'|'`` or ``'('``, are special. Special
 characters either stand for classes of ordinary characters, or affect
 how the regular expressions around them are interpreted.
 
-Repetition qualifiers (``*``, ``+``, ``?``, ``{m,n}``, etc) cannot be
+Repetition quantifiers (``*``, ``+``, ``?``, ``{m,n}``, etc) cannot be
```
Comment (serhiy-storchaka) [inline Doc/library/re.rst]:
```suggestion
Repetition operators or quantifiers (``*``, ``+``, ``?``, ``{m,n}``, etc) cannot be
```

File: Doc/howto/regex.rst [hunk 5]
```diff
@@ -230,13 +230,13 @@ while ``+`` requires at least *one* occurrence.  To use a similar example,
 ``ca+t`` will match ``'cat'`` (1 ``'a'``), ``'caaat'`` (3 ``'a'``\ s), but won't
 match ``'ct'``.
 
-There are two more repeating qualifiers.  The question mark character, ``?``,
+There are two more repeating operators or quantifiers.  The question mark character, ``?``,
```
Comment (felixxm) [inline Doc/howto/regex.rst:233]: Wrap at 80 chars.

File: Doc/library/re.rst [hunk 6]
```diff
@@ -87,7 +87,7 @@ Some characters, like ``'|'`` or ``'('``, are special. Special
 characters either stand for classes of ordinary characters, or affect
 how the regular expressions around them are interpreted.
 
-Repetition qualifiers (``*``, ``+``, ``?``, ``{m,n}``, etc) cannot be
+Repetition operators or quantifiers (``*``, ``+``, ``?``, ``{m,n}``, etc) cannot be
```
Comment (felixxm) [inline Doc/library/re.rst:90]: Wrap at 80 chars.

File: Doc/library/re.rst [hunk 7]
```diff
@@ -146,10 +146,10 @@ The special characters are:
    single: ??; in regular expressions
 
 ``*?``, ``+?``, ``??``
-   The ``'*'``, ``'+'``, and ``'?'`` qualifiers are all :dfn:`greedy`; they match
+   The ``'*'``, ``'+'``, and ``'?'`` quantifiers are all :dfn:`greedy`; they match
```
Comment (felixxm) [inline Doc/library/re.rst:149]: Wrap at 80 chars.

**Review by felixxm — APPROVED** [review]
@serhiy-storchaka Thanks :+1:

File: Doc/howto/regex.rst (hunk 5 above)
Comment (serhiy-storchaka) [reply Doc/howto/regex.rst:233]: I think that in this case it is better to keep a line slightly longer that 80 characters than split it on two very short lines or reformat the whole paragraph. PEP 8 allows some liberty.

Comment (felixxm) [reply Doc/howto/regex.rst:233]: I wrongly assumed that in Python docs (as we do in Django docs) re-wrapping is stricter. TIL. Thanks :+1:

Comment (serhiy-storchaka) [reply Doc/howto/regex.rst:233]: Small compromises are possible. If it would be a new text, it would be stricter.

File: Doc/library/re.rst (hunk 6 above)
Comment (serhiy-storchaka) [reply Doc/library/re.rst:90]: It is not PEP 8, but some other PEP for documentation. In any case a common sense is the main principle.

**Pull request merged**